pkg/yang/entry.go

```diff
@@ -71,9 +71,12 @@ type deviationPresence struct {
 	hasMaxElements bool
 }
 
-// An Entry represents a single node (directory or leaf) created from the
-// AST.  Directory entries have a non-nil Dir entry.  Leaf nodes have a nil
-// Dir entry.  If Errors is not nil then the only other valid field is Node.
+// An Entry represents a single node (directory or leaf) created from the AST
```

Comment:
Directory, leaf or leaf-list? I wonder whether it's best to say `schema tree node, which can be a directory (containing a subtree) or a leaf node (which contains YANG types that have no children, e.g., leaf, leaf-list)`.

Reply:
Done.